accounts/tests.py

```diff
@@ -1,47 +1,162 @@
+from django.urls import reverse
 from django.test import TestCase
-
+from .models import User
 
 class TestSignUpView(TestCase):
     def test_success_get(self):
-        pass
+        response = self.client.get(reverse('accounts:signup'))
+        self.assertEqual(response.status_code,200)
+        self.assertTemplateUsed(response, 'accounts/signup.html')
 
     def test_success_post(self):
-        pass
+        post ={
+            'email' : 'test@example.com',
+            'username' : 'test',
+            'password1' : 'goodpass',
+            'password2' : 'goodpass',
+        }
+        response = self.client.post(reverse('accounts:signup'), post)
+        self.assertEqual(response.status_code, 302)
+        self.assertTrue(User.objects.filter(username = 'test',email = 'test@example.com').exists())#ユーザーが追加されたかを確認　ついでに正しく登録されているかを確認

```

Comment:
```suggestion
        self.assertTrue(User.objects.filter(username='test', email='test@example.com').exists())#ユーザーが追加されたかを確認　ついでに正しく登録されているかを確認
```
通常の変数宣言の時の=などは両側にスペース空けるのですが、キーワード引数での=は開けなくていいです。これはpythonのコード規則であるpep8でそうなっていますので確認してみてください。
ですので、他の箇所もpep8に従って直してみてください